Author: Shawn Huang <phuang@redhat.com>
Source: FC14, Fri Nov 24 2006, 1.4.5-4
Description: turn off key snooper in im-scim to avoid crashes when clicking during scim gtkimm preedit 
--- a/extras/gtk2_immodule/gtkimcontextscim.cpp
+++ b/extras/gtk2_immodule/gtkimcontextscim.cpp
@@ -308,7 +308,7 @@
 
 static bool                                             _on_the_spot                = true;
 static bool                                             _shared_input_method        = false;
-static bool                                             _use_key_snooper            = true;
+static bool                                             _use_key_snooper            = false;
 
 // A hack to shutdown the immodule cleanly even if im_module_exit () is not called when exiting.
 class FinalizeHandler
